### ISSUE TITLE
[Improvement] Simplify Tadpoles login flow for GMail auth accounts

### ISSUE THREAD

**Issue by s0rcy** [leocov-dev/tadpoles-backup#44]
For Tadpoles users with Gmail addresses / Google auth, an alternative to using the bookmark and javascript manipulation is to trigger the "Forgot password" request flow to tadpoles.com directly with curl.

A Tadpoles password reset can be triggered with the following curl command (replace email address):

```
curl -i -s -k -X $'POST' \
    -H $'Host: www.tadpoles.com' -H $'Content-Type: application/x-www-form-urlencoded; charset=UTF-8' \
    --data-binary $'email=GMAIL_ADDRESS_HERE@gmail.com&app=parent&service=tadpoles' \
    $'https://www.tadpoles.com/auth/forgot'
```

This should send a tadpoles.com password reset link to the gmail email address, allowing the user to set a site-specific password.

**Comment by leocov-dev** [comment]
I would absolutely love to remove all of the JavaScript stuff from this project. It's a huge kludge.
The host header manipulation is what's necessary for the request not to fail.
I can't actually rely on an external tool however, windows users are not likely to have curl.
It should be possible to make the same formatted call from Go's api client as well. I will re-check.

**Comment by leocov-dev** [comment]
will be resolved with:
https://github.com/leocov-dev/tadpoles-backup/pull/45

**Comment by leocov-dev** [comment]
this change will be available with release `v2.1.0`
